### PR TITLE
pin DiffEqBase

### PR DESCRIPTION
**Pull request by costachris**
## Checklist before requesting a review / merging
- [x] I have formatted the code using `julia --project=.dev .dev/climaformat.jl .`
- [x] I have updated all test manifests using `julia --project .dev/up_deps.jl .` with Julia 1.7.3.
- [x] If core features are added, I have added appropriate test coverage.
- [x] If new functions and structs are added, they are documented through docstrings.

### CODE REVIEW

**Review by ilopezgp — APPROVED** [review]
LGTM

File: integration_tests/Project.toml [hunk 1]
```diff
@@ -35,3 +36,7 @@ Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
 Thermodynamics = "b60c26fb-14c3-4610-9d3e-2d17fe7ff00c"
 TurbulenceConvection = "8e072fc4-01f8-44fb-b9dc-f9336c367e6b"
 UnPack = "3a884ed6-31ef-47d7-9d2a-63182c4928ed"
+
+
+[compat]
+DiffEqBase = "= 6.89.0"
```
Comment (haakon-e) [inline integration_tests/Project.toml:42]: we should add a newline here in a future PR

Comment (haakon-e) [reply integration_tests/Project.toml:42]: added to #180

**Pull request merged**